src/vulkan/vulkan_renderer.cpp

```diff
@@ -38,6 +46,14 @@ namespace IC {
     }
 
     void VulkanRenderer::DrawFrame() {
+        ImGui_ImplVulkan_NewFrame();
+        ImGui_ImplGlfw_NewFrame();
+        ImGui::NewFrame();
+        for (auto function : imGuiFunctions) {
+            function();
+        }
+        ImGui::Render();
```

Comment:
From what I remember ImGui wants to be rendered after everything else. Is that still the case or does it not matter if we do it first here?